### DISCUSSION TITLE
Кнопка создания новой категории таблиц

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#380] [ОШБК!] [answered]
При создании таблицы можно указать категорию. Над категорией есть 2 кнопки: левая редактировать, правая добавить
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/36eeedc9-1517-4c27-99ff-2aa8a34c0a8e)

По факту обе кнопки делают одно и то же - редактирование, т.е. создать новую категорию отсюда нельзя

**Comment by ttm-alexeypolunin** [comment]
А не только здесь — везде для такого типа кнопок.

**Reply by simplepersonru** [reply]
так и задумано? Я человек простой, вижу багу - пишу запрос)

**Reply by ttm-alexeypolunin** [reply]
нет, не задумано

**Reply by ttm-alexeypolunin** [reply, marked as answer]
**fixed**
